news/2025/2025-11-11-resurrection-cup-2025-results.md

```diff
@@ -0,0 +1,456 @@
+
+The feedback has been mainly positive despite the struggles during pooling. What do you consider as key thing(s) in such situations or, rather what advice would you give to people who'd pump out similar amounts of original projects in the osu! tournament community per iteration?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/5596337" style="background-image: url('/wiki/shared/avatars/Daycore.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-bn" href="https://osu.ppy.sh/users/5596337">Daycore</a></p>
+
+Have fun and remember that all of the maps you work with will be played by the players. You won't make a good 'product' without thinking of it's consumers. Players are a fuel for tournaments, so we've got to provide them with a fun experience to receive positive feedback.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11681430" style="background-image: url('/wiki/shared/avatars/Yumerios.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/11681430">Yumerios</a></p>
+
+I like to think of it like a puzzle, your custom maps are like pieces you need to place to assemble and give importance to create the final image, what's interesting with custom maps is that you can still move them around, talk with the mapper for some changes to make it fit one slot, so if a specific maps is not the piece that fills one part of the puzzle, then it can be the one filling an other part of it. The most important thing to keep in mind is to communicate and be clear on what is necessary, while keeping a good vibe in the team obviously
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/12840110" style="background-image: url('/wiki/shared/avatars/Phreel-2.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/12840110">Phreel</a></p>
+
+You've finally got a ResCup Grandfinals Tiebreaker to work on, what were your initial thoughts about [Rigid Memory](https://osu.ppy.sh/beatmapsets/2414048)? Was it hard meeting the 9-star threshold?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11681430" style="background-image: url('/wiki/shared/avatars/Yumerios.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/11681430">Yumerios</a></p>
+
+I think I expected something... different ? But it ended up being way better than everything I could imagine, like this song was so good and gave me such a big flow of ideas on how I could map it. I really am proud of the final result, huge thanks to everyone involved in the song.
+
+Only thing I would've wanted is for something that had a bit of a higher bpm, because I was not sure before the very end of the map that I was gonna hit the 9-stars threshold. Not a big deal tho.
```

Comment:
```suggestion
Only thing I would've wanted is for something that had a bit of a higher bpm, because I was not sure before the very end of the map that I was gonna hit the 9-star threshold. Not a big deal tho.
```